### DISCUSSION TITLE
1,2,3,4,5,6,7,8

### DISCUSSION THREAD

**Discussion by J-Patlmx** [community/community#54976] [GitHub Education]
8 tries and each time the same impossible to validate my student status, do you think that a human can contact me to solve this problem? my online university my establishment is not next to me!

**Comment by samus-aran** [comment]
Hi @J-Patlmx 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

I would also suggest you ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
